packages/backend/src/gerrit.ts

```diff
@@ -0,0 +1,119 @@
+import fetch from 'cross-fetch';
+import { GerritConfig } from './schemas/v2.js';
+import { AppContext, GitRepository } from './types.js';
+import { createLogger } from './logger.js';
+import path from 'path';
+import { measure, marshalBool, excludeReposByName, includeReposByName } from './utils.js';
+
+// https://gerrit-review.googlesource.com/Documentation/rest-api.html
+interface GerritProjects {
+   [projectName: string]: GerritProjectInfo;
+}
+
+interface GerritProjectInfo {
+   id: string;
+   state?: string;
+   web_links?: GerritWebLink[];
+}
+
+interface GerritWebLink {
+   name: string;
+   url: string;
+}
+
+interface GerritBranch {
```

Comment:
nit: I think this is unused

_aside: will add linter support to the backend s.t., we can automate this stuff_

Reply:
dones